src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,131 @@
+package duke;
+
+import duke.task.*;
```

Comment:
Perhaps importing the classes individually instead of using a wildcard would be better to adhere to the coding standard.